src/veritysetup/veritysetup.c

```diff
@@ -224,6 +228,31 @@ static int parse_options(const char *options) {
                         r = free_and_strdup(&arg_hash, val);
                         if (r < 0)
                                 return log_oom();
+                } else if ((val = startswith(word, "fec-device="))) {
+
+                        r = free_and_strdup(&arg_fec_what, val);
+                        if (r < 0)
+                                return log_oom();
+                } else if ((val = startswith(word, "fec-offset="))) {
+                        uint64_t off;
+
+                        r = safe_atou64(val, &off);
```

Comment:
parse_bytes()?


i mean, these are offsets in bytes, no? I'd jst use parse_bytes() for those then too, then

Reply:
done.

Hum, do you mean `parse_size()`? there is no `parse_bytes()` function.

Reply:
yuppp, sorry